src/Map.js

```diff
@@ -246,22 +260,22 @@ export default function Map (props) {
       }
       const {
         encointerCeremonies: {
-          participantCount: getParticipantCount,
+          assignmentCounts: getAssignmentCount,
           meetupCount: getMeetupCount
         }
       } = api.query;
       // Previous Phases of current Ceremony
       for (let oldPhase = 0; oldPhase < phase; oldPhase++) {
         cids.forEach(cid => {
           const communityCeremony = new CommunityCeremony(api.registry, [cid, ceremony]);
-          const getters = [getParticipantCount, getMeetupCount];
+          const getters = [getAssignmentCount, getMeetupCount];
           const getter = getters[oldPhase];
-          debug && console.log('hist ', bs58.encode(cid), ceremony.toNumber(), oldPhase);
-          getter(communityCeremony).then((_) => dispatch({
+          debug && console.log('hist ', cid, ceremony.toNumber(), 'oldphase', oldPhase);
+          getter(communityCeremony).then((assignmentOrMeetupCount) => dispatch({
             type: setters[oldPhase],
             payload: {
-              cid: bs58.encode(cid),
-              count: _.toNumber()
+              cid: communityIdentifierToString(cid),
+              count: (oldPhase === 0) ? sumUp(assignmentOrMeetupCount.toJSON()) : assignmentOrMeetupCount.toNumber()
```

Comment:
The reason why I added this comparison is because the .toNumber() function failed everytime we were in phase 0. The reason is, that the other phases (meetup/attestation) return numbers, but assignements return {bootrappers: number, endorsees: number, newbies: number}..
See the difference?

Reply:
Ah ok, i did not really get that piece of code. Now I understand. 👍

Reply:
not so hirsch afterall...